README.markdown

```diff
@@ -10,220 +10,134 @@ There are two ways in which you can install OneZoom on a personal computer: full
-5. Note that the normal `minlife.html` file will use local versions of data files and the javascript treeviewer, but will get API information form the OneZoom website, *and* also use the OneZoom website as the source for the html page which embeds the viewer. For developers only, who may wish to create a minlife version not only using modified javascript in the treeviewer but also with bespoke html, you can run `grunt partial-local-install`. This is much more effort since it requires you to set up a full installation (as below) before creating the minlife scripts, but once created, the files in `static` will be enough for other users to view (and test) your modifications.
-
-### For a full installation (recommended):
-	
-1. Install a source code version of [web2py](http://www.web2py.com), placing your [OZtree repository](https://github.com/OneZoom/OZtree) within the web2py `applications` directory.
-2. Install command-line software by running `npm install -g grunt-cli` (you may need to do all this with administrator privileges).
-3. Run `npm install` from within the OZtree folder you moved in step 1. then run `grunt dev` (or `grunt prod` if in production mode) - see *"[Building the OneZoom tree viewer](#building-the-onezoom-tree-viewer)"*.
-3. [Install](http://dev.mysql.com/downloads/mysql/) & start MySQL, then create a new database (see *"[Setting up the database backend](#setting-up-the-database-backend)"*)
-4. Create a appconfig.ini file in `OZtree/private`, with `migrate=1` and which references this database with the appropriate username and password. We also recommend copying the `routes.py` file from `OZtree/_COPY_CONTENTS_TO_WEB2PY_DIR` to the top level of your web2py installation - see *"[Web2py installation](#web2py-installation)"*
-5. Fire up a temporary web2py server and visit the main page to create the (empty) database tables - see *"[Starting and shutting down web2py](#starting-and-shutting-down-web2py)"*
-6. Load up data into the tables: first create a user and assign it a 'manager' role in the `auth_` tables using the web2py database admin pages, then load the other tables using data from the original OneZoom site (e.g. sent to you via file transfer) - see *"[Filling the database](#filling-the-database)"*.
-7. Optimise your installation:
-	* create indexes on the tables by running the SQL script in `OZtree/OZprivate/ServerScripts/SQL/create_db_indexes.sql`. You can do this, for example, by running `SOURCE /path/to/OZtree/OZprivate/ServerScripts/SQL/create_db_indexes.sql` within a mysql client.
-	* set `is_testing = False` in `models/db.py` and `migrate=0` in appconfig.ini.
 
+For any installation, OneZoom requires node & python (3.11).
```

Comment:
I thought we had settled on 3.10 for now.

Reply:
Hrm [I took @hyanwong's last comment here](https://github.com/OneZoom/OZtree/issues/72#issuecomment-1864335699) to imply 3.11 was the way?

Is there another thread discussing this?

Reply:
Yes there is, https://github.com/OneZoom/OZtree/pull/699. But there doesn't seem to be much rationale of 3.10 vs 3.11. Anyways, it's not a big deal.